base/stats.go

```diff
@@ -85,6 +85,7 @@ const (
 	StatAddedVersion3dot1dot3dot1 = "3.1.3.1"
 	StatAddedVersion3dot1dot4     = "3.1.4"
 	StatAddedVersion3dot2dot0     = "3.2.0"
+	StatAddedVersion3dot2dot1     = "3.2.1"
```

Comment:
Will file a backport ticket now so we don't forget.

Reply:
This is part of the memory based rev cache epic that has a backport epic with it [CBG-3825](https://issues.couchbase.com/browse/CBG-3825) 
But I agree that this should have its own backport ticket as some of the memory based work is up in the air waiting on perf results.

Reply:
Ah sorry, I forgot we were tracking the backport as the whole epic